src/pudl/transform/params/ferc1.py

```diff
@@ -4854,6 +4854,7 @@
                 "other_sales_to_public_authorities_axis",
                 "sales_to_railroads_and_railways_axis",
                 "interdepartmental_sales_axis",
+                "provision_for_rate_refunds_axis",
```

Comment:
this got added in bc this is effectively a new table that we are now processing as a part of the rate schedule table bc of fixing that typo